lib/LaTeXML/Engine/TeX_Math.pool.ltxml

```diff
@@ -700,6 +700,30 @@ DefConstructor('\mathinner Digested', sub { adjustMathRole('ATOM',  @_); });
 # \left     c  makes TeX calculate the size of the delimiter needed at the left of a subformula.
 # \right    c  makes TeX calculate the size of the delimiter needed at the right of a subformula.
 
+# Read a Delimiter, for \left,\right, but also \big, etc
+# Formally a delimiter is either a token, or \delimiter<number> or maybe \radical<number>,
+# but we don't actually restrict to those.
+# Note that \big expands into \left, unwrapping {} once; only the first atom gets big.
+DefParameterType('TeXDelimiter', sub {
+  no warnings 'recursion';
+  my ($gullet) = @_;
+  $gullet->skipFiller;          # Skip spaces and \relax
+  my $token = $gullet->readXToken(0);
+  if ($token && $token->getCatcode == CC_BEGIN) { # Unwrap once; re-read token
+    $gullet->unread($gullet->readBalanced(1));
+    $gullet->skipFiller;
+    $token = $gullet->readXToken(0); }
+  # Replace "." by special hint.
+  $token = T_CS('\lx@delimiterdot') if (! defined $token) || (ToString($token) eq '.');
+  my ($delim) = $STATE->getStomach->invokeToken($token);
+  return $delim; },
+  undigested => 1,                         # since _already_ digested.
+  reversion  => sub { Revert($_[0]); });   # shouldn't need braces
+
+DefConstructor('\lx@delimiterdot', "<ltx:XMHint/>",
```

Comment:
Just a side-remark:

This reminded me I had an old confusion about the empty XMhint elements. How do we know what they hint? Do all hints hint at the same thing, etc.